src/testdata/private_key_test.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
can you include the command you used to generate the keys in the commit message?

Reply:
Done